### PR TITLE
contributing...

### PR DESCRIPTION
**Pull request by mdipenta**
I've developed this new feature and I want it to be reviewed...

### CODE REVIEW

File: newft.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (maxdipenta) [inline newft.txt:1]: Here there should be a comment

Comment (maxdipenta) [reply newft.txt:1]: A comment is missing. Also you should run checkstyle

**Review by maxdipenta — APPROVED** [review]

**Review by mdipenta — COMMENTED** [review]
change done

**Comment by mdipenta** [comment]
Pull request is now merged

**Pull request closed without merging**